VRCFaceTracking/Params/Expressions/Legacy/Lip/LipShapeConversion.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Linq;
+
+namespace VRCFaceTracking.Params.Lip
+{
+    internal interface ICombinedShape
```

Comment:
All shapes that implement ICombinedShape would become obsolete too since only the old SRanipal shapes use it, (Unified Expressions' combined shapes is purely controlled with delegates and doesn't touch this class)